src/benchmark_runner.cc

```diff
@@ -482,6 +505,10 @@ void BenchmarkRunner::DoOneRepetition() {
   run_results.non_aggregates.push_back(report);
 
   ++num_repetitions_done;
+
+  if (profiler_manager != nullptr) {
```

Comment:
nit: can we run this right after the memory manager and before the reporting?

Reply:
Sure